helm-chart/binder-launches/templates/configmap.yaml

```diff
@@ -4,18 +4,20 @@ metadata:
   name: {{ .Release.Name }}-configmap
 data:
   # `|-` is for declaring a multi-line string
+  # To check boolean values explicity, we use
+  # (kindIs "invalid") to check if the value is nil.
   config.js: |-
     const config = {
     {{- if .Values.baseUrl }}
     baseUrl: {{ .Values.baseUrl | quote }},
     {{- end }}
-    {{- if .Values.port }}
-    port: {{ .Values.port }},
+    {{- if .Values.db.port }}
+    port: {{ .Values.db.port }},
     {{- end }}
```

Comment:
I just realised that this config is unnecessary here. This port is the application port, it is always 3001 and it is hardcoded in deployment.yaml and service.yaml. Could you delete these 3 lines for port configuration? Thank you for pointing that out! :)

Reply:
done :)